bin/i18n/sync-out.rb

```diff
@@ -390,6 +390,23 @@ def distribute_translations(upload_manifests)
       sanitize_file_and_write(loc_file, destination)
     end
 
+    ### Merge ml-playground datasets into apps' ailab JSON
```

Comment:
Piggybacking these translations onto the existing ailab locale translations that we pass into the Ailab component. The unique thing here is there usually isn't nested keys in those files, but testing locally shows there is no issue with it.

Reply:
With the translated locale files written to with these updates, I considered what if these "piggybacked" translations are then synced in and create _duplicate_ translation strings in Crowdin. From my understanding we don't need to worry about this scenario because we [only sync in the en_us files](https://github.com/code-dot-org/code-dot-org/blob/sync-in-ai-lab-datasets/bin/i18n-codeorg/in.sh#L38) that are [untouched](https://github.com/code-dot-org/code-dot-org/blob/sync-in-ai-lab-datasets/bin/i18n/sync-out.rb#L353) by this sync out logic.